converter/convert_test.go

```diff
@@ -1,17 +1,17 @@
-package oci
+package converter
 
 import (
-	"testing"
-
-	"github.com/docker/cnab-to-oci/test"
+	"github.com/docker/cnab-to-oci/tests"
 	"github.com/docker/distribution/manifest/schema2"
 	"github.com/docker/distribution/reference"
 	ocischemav1 "github.com/opencontainers/image-spec/specs-go/v1"
 	"gotest.tools/assert"
+
+	"testing"
```

Comment:
this re-ordering is weird... `testing` should be first.

Reply:
Yep. Looks like it passes the linter. 
But I just executed a `make format` and it was replaced :man_facepalming: